Add missing `BuyRevealedMine` reaction in `UserRequestType`

Title is self explanatory.

Currently server throws this error:
```
[2023-04-20 17:53:45] [ERROR] [ReactionManager.java:149] Class `com.github.splendor_mobile_game.websocket.handlers.reactions.BuyRevealedMine` has ReactionName=BUY_REVEALED_MINE, but no mathing variant in UserRequestType has been found!
```

--- a/src/main/java/com/github/splendor_mobile_game/websocket/handlers/UserRequestType.java
+++ b/src/main/java/com/github/splendor_mobile_game/websocket/handlers/UserRequestType.java
@@ -9,5 +9,6 @@ public enum UserRequestType {
     START_GAME,
     DEBUG_GET_RANDOM_CARD,
     LEAVE_ROOM,
-    GET_TOKENS
+    GET_TOKENS,
+    BUY_REVEALED_MINE
 }

